diff --git a/lib/neutron-legacy b/lib/neutron-legacy
index 016df431..f683602f 100644
--- a/lib/neutron-legacy
+++ b/lib/neutron-legacy
@@ -689,7 +689,7 @@ function _move_neutron_addresses_route {
             IP_DEL="sudo ip addr del $IP_BRD dev $from_intf"
             IP_REPLACE="sudo ip addr replace $IP_BRD dev $to_intf"
             IP_UP="sudo ip link set $to_intf up"
-            if [[ "$af" == "inet" ]]; then
+            if [[ "$af" == "inet" ]] && !is_ubuntu; then
                 IP=$(echo $IP_BRD | awk '{ print $1; exit }' | grep -o -E '(.*)/' | cut -d "/" -f1)
                 ARP_CMD="sudo arping -A -c 3 -w 5 -I $to_intf $IP "
             fi
