diff --git a/lib/neutron b/lib/neutron
index 8708bf43..e3fe7b05 100644
--- a/lib/neutron
+++ b/lib/neutron
@@ -747,7 +747,7 @@ function _move_neutron_addresses_route {
             IP_DEL="sudo ip addr del $IP_BRD dev $from_intf"
             IP_REPLACE="sudo ip addr replace $IP_BRD dev $to_intf"
             IP_UP="sudo ip link set $to_intf up"
-            if [[ "$af" == "inet" ]]; then
+            if [[ "$af" == "inet" ]] && !is_ubuntu; then
                 IP=$(echo $IP_BRD | awk '{ print $1; exit }' | grep -o -E '(.*)/' | cut -d "/" -f1)
                 ARP_CMD="sudo arping -A -c 3 -w 5 -I $to_intf $IP "
             fi
